privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -59,8 +59,11 @@ <h1>{% ftlmsg 'multi-part-onboarding-premium-get-domain' %}</h1>
                         {% endif %}
                         
                         <div class="c-premium-onboarding-action-completed {% if user_profile.subdomain %} is-visible {% endif %}">
-                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
-                            <samp class="js-premium-onboarding-domain-registration-preview">{{ user_profile.custom_domain }}</samp>
+                            <div class="c-premium-onboarding-subdomain-set-module">
+                                <p class="c-premium-onboarding-label">{% ftlmsg 'profile-label-domain' %}</p>
+                                <span>@</span><p class="c-premium-onboarding-subdomain js-premium-onboarding-domain-registration-preview">{{ user_profile.subdomain }}</p>
```

Comment:
Suggestion: If we're previewing the name, I recommend we stick with the [`samp` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/samp) like we're using else where.

Reply:
Done!